marathon.json

```diff
@@ -0,0 +1,10 @@
+{
+    "id": "hdfs",
+    "cmd": "cd hdfs* && ./bin/hdfs-mesos",
+    "uris": [
+        ""
+    ],
+    "cpus": 0.1,
+    "mem": 128.0,
```

Comment:
Is this a sufficient amount of cpu/memory for the scheduler? I know it's pretty lightweight, but I just wanted to verify that these values make sense if isolation is actually enabled.